pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -66,22 +66,6 @@ var (
 	PreflightInProgress PreflightStatus = ""
 	PreflightSuccess    PreflightStatus = "successful"
 	PreflightFail       PreflightStatus = "failed"
-
-	OperatorVersionAMQStreams            = "1.1.0"
```

Comment:
I think this is still needed by Damien's changes https://github.com/integr8ly/integreatly-operator/pull/217/files#diff-bff50e11f27449f5ea8f93636ade1090L70-L84

fyi @damienomurchu

Reply:
Yeah, looks like it. I'll rebase with master to add it back it when Damien's PR is merged

Reply:
that's a pity. Is there no way to work this out dynamically? From the manifest files or something? Would prefer to not have to maintain these values if possible.

Reply:
@philbrookes Yeah, I think it's possible actually since I pull out the version from the manifest `package.yaml` file. Although I'm not sure is @damienomurchu good with this